contrib/easyconfig-templates/2022a/Bundle-common-components.eb.tmpl

```diff
@@ -0,0 +1,52 @@
+# Template for a bundle of components from a common organisation and using the same easyblock on GCCcore 11.3.0
+easyblock = 'Bundle'
+
+name = 'SoftwareName'
+version = '0.0.0'
+
+homepage = 'https://www.domain.org'
+description = """
+Description in 80 chars long column
```

Comment:
I thought the informal rule was keep to around 100 characters for description?

Reply:
I prefer 80 because that is easier to read in the final module file. 100 characters is a rather long line.

Moreover, 80 is the default in `vi` to cut text in columns.